source/magpie/ops.d

```diff
@@ -0,0 +1,76 @@
+module magpie.ops;
+
+import std.algorithm: map, reduce;
+
+double count(T)(T[] arr)
```

Comment:
you should try to make aggregate work directly with phobos (and mir's if you can) algorithms directly without the need for this shim.